packages/test-utils/src/run.ts

```diff
@@ -1,31 +1,36 @@
-import { Entity, EntityManager, isEntity } from "joist-orm";
+import { Entity, EntityManager, isDefined, isEntity } from "joist-orm";
 import { fail } from "joist-utils";
 import { Context } from "./context";
 
 type MaybePromise<T> = T | Promise<T>;
 
 export type ContextFn<C> = (ctx: C) => MaybePromise<C>;
 
+type RunOptions = {
+  expectUnsavedEntities?: boolean;
```

Comment:
Thought: maybe `skipNewEntities`?

I'm also kinda wondering if, if this is for testing dry runs, should we just calling this like `dryRun: true` or `reloadEntities: false`, and have it skip the load "translate loaded entities" entirely?

Reply:
I also would prefer a less verbose name.  `dryRun` seems more natural to me, imo.